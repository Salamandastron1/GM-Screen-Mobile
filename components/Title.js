import React from 'react';
import { View, Text } from 'react-native';
import PropTypes from 'prop-types';

const Title = ({ children }) => (
  <View style={styles.viewStyle}>
    <Text style={styles.textStyle}>
      {children}
    </Text>
  </View>
);

const styles = {
  textStyle: {
    color: '#7A000D',
    fontSize: 70,
    alignSelf: 'center',
    fontWeight: '600',
  },
  viewStyle: {
    height: 60,
    borderRadius: 2,
    alignSelf: 'stretch',
    justifyContent: 'center',
    margin: 20,
    height: null,
  }
};

Title.propTypes = {
  children: PropTypes.string.isRequired,
};

export default Title;
