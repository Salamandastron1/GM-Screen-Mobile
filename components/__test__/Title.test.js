/* eslint-disable */

import React from 'react';
import { shallow } from 'enzyme';
import Title from '../Title';

it('should match the shapshot', () => {
  const tree = shallow(<Title />);
  expect(tree).toMatchSnapshot();
});