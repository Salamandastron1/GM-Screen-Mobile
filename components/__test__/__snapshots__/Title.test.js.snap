// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the shapshot 1`] = `
<Component
  style={
    Object {
      "alignSelf": "stretch",
      "borderRadius": 2,
      "height": null,
      "justifyContent": "center",
      "margin": 20,
    }
  }
>
  <Component
    style={
      Object {
        "alignSelf": "center",
        "color": "#7A000D",
        "fontSize": 70,
        "fontWeight": "600",
      }
    }
  />
</Component>
`;
